packages/react-components/react-accordion/src/components/Accordion/Accordion.test.tsx

```diff
@@ -8,7 +8,12 @@ describe('Accordion', () => {
     Component: Accordion,
     displayName: 'Accordion',
     // Accordion does not have own styles
-    disabledTests: ['make-styles-overrides-win', 'consistent-callback-args'],
+    disabledTests: ['make-styles-overrides-win', 'consistent-callback-args-legacy'],
+    testOptions: {
+      'consistent-callback-type': {
+        ignoreProps: ['onToggle'],
```

Comment:
why are some handlers being ignored ? what's the plan here , will it be fixed in a follow up PR ?

Reply:
The existing callbacks will be verified by the legacy test `consistent-callback-args-legacy`.
The newly added callbacks will be verified by `consistent-callback-type`

Reply:
so existing v9 packages (this one in particular), doesnt contain any old callback types ?